packages/fxa-content-server/tests/functional/pages.js

```diff
@@ -47,9 +47,6 @@ var pages = [
   'reset_password_verified',
   'settings',
   'settings/avatar',
-  'settings/avatar/camera',
-  'settings/avatar/change',
-  'settings/avatar/crop',
```

Comment:
These routes don't exist anymore. I don't think we need to add redirect here because I don't believe any external apps link directly to these pages.